### ISSUE TITLE
Achieve Accessibility Score of 95%

### ISSUE THREAD

**Issue by jaypeasee** [jaypeasee/overlook-hotel#10]
#### Regularly do `lighthouse` score reviews from `Dev Tools` to evaluate and cleanup accessibility errors as they come up from new `HTML` creations. 

When the the project is turned in, the page should be at 95% or higher.
